resources/lang/pt_pt.json

```diff
@@ -1,44 +1,5 @@
-    "Log in": "Login",
-    "E-Mail Address": "Endereço de correio eletrónico",
-    "Password": "Palavra-passe",
-    "Remember Me": "Lembrar-se de mim",
-    "Forgot Your Password?": "Esqueceu a sua palavra-passe?",
-    "Register": "Registo",
-    "Name": "Nome",
-    "Confirm Password": "Confirmar palavra-passe",
-    "Reset Password": "Redefinir palavra-passe",
-    "Reset Password Notification": "Redefinir notificação de palavra-passe",
-    "You are receiving this email because we received a password reset request for your account.": "Está a receber este email porque recebemos um pedido de redefinição de palavra-passe da sua conta.",
-    "This password reset link will expire in :count minutes.": "Este link de redefinição de palavra-passe vai expirar em :count minutos.",
-    "If you did not request a password reset, no further action is required.": "Se não solicitou uma redefinição de palavra-passe, não é necessária qualquer ação adicional.",
-    "Regards": "Cumprimentos",
-    "Whoops!": "Opa!",
-    "Hello!": "Olá!",
-    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser: [:actionURL](:actionURL)": "Se estiver com problemas em clicar no botão \":actionText\", copie e cole o link abaixo\nno seu navegador de internet: [:actionURL] (:actionURL)",
-    "Send Password Reset Link": "Enviar link de redefinição de palavra-passe",
-    "Logout": "Sair",
-    "Verify Email Address": "Verificar endereço de correio eletrónico",
-    "Please click the button below to verify your email address.": "Por favor, clique no botão abaixo para verificar o seu endereço de correio eletrónico.",
-    "If you did not create an account, no further action is required.": "Se não criou uma conta, não é necessária qualquer ação adicional.",
-    "Verify Your Email Address": "Verifique o seu endereço de correio eletrónico",
-    "A fresh verification link has been sent to your email address.": "Um novo link de verificação foi enviado para o seu endereço de correio eletrónico.",
-    "Before proceeding, please check your email for a verification link.": "Antes de prosseguir, verifique no seu correio eletrónico por um link de verificação.",
-    "If you did not receive the email": "Se não recebeu o correio eletrónico",
-    "click here to request another": "clique aqui para solicitar outro",
-    "Back to login": "Voltar ao login",
-    "Don't have an account?": "Não tem uma conta?",
-    "Already have an account?": "Já possui uma conta?",
-    "From the lovely folks at": "Das pessoas adoráveis da",
-    "Source & Roadmap": "Código fonte & roadmap",
-    "Table of contents": "Índice",
     "All resources": "Todos os recursos",
     ":locale resources": "Recursos em :locale",
-    "English and :locale resources": "Recursos em inglês e :locale"
+    "English and :locale resources": "Recursos em Inglês e :locales"
```

Comment:
After the colon, this is the name of a variable that cannot be translated.
So instead of ```:locales``` you should leave  ```:locale```

Reply:
Yah, @olsza is correct here.

Reply:
Fixed